src/repeat.js

```diff
@@ -120,6 +120,9 @@ export class Repeat {
 
     let items = this.items;
     this.strategy = this.strategyLocator.getStrategy(items);
+    if (!this.strategy) {
+      return;
```

Comment:
That sounds reasonable to me. Let's do it. @gingerik Would you be interested in amending your PR to throw an error  based on @jdanyow 's recommendations? If you aren't comfortable with that we could take this PR and then make the change ourselves. Let me know.